client/js/components/GameView/GameDialogs.vue

```diff
@@ -121,7 +122,8 @@ export default {
         this.resolvingSeven &&
         this.topCard.rank === 11 &&
         (!this.secondCard || this.secondCard.rank === 11) &&
-        (this.opponentPointTotal === 0 || this.opponentQueenCount > 0)
+        (this.opponentPointTotal === 0 || this.opponentQueenCount > 0) &&
+        this.isPlayersTurn === true
```

Comment:
Then we could remove the turn check here
```suggestion
        (this.opponentPointTotal === 0 || this.opponentQueenCount > 0)
```